lib/node/NodeEnvironmentPlugin.js

```diff
@@ -18,12 +19,31 @@ class NodeEnvironmentPlugin {
 	 * @returns {void}
 	 */
 	apply(compiler) {
+		const isMemoryFS = compiler.options.memory ? true : false;	
```

Comment:
Can be simplified to `Boolean(compiler.options.memory)`? (or just using `compiler.options.memory` as a falsy or not in conditions)